### PR TITLE
Fix to make q_plot work with matplotlib 3.0.0+

### PR DESCRIPTION
**Pull request by FlorianvdEnt**

### CODE REVIEW

File: qscripts-cli/q_plot.py [hunk 1]
```diff
@@ -399,15 +403,28 @@ def on_resize(self,event):
...
-            (ubuntu)    $ sudo apt-get install python-matplotlib
-            (mac)       $ sudo brew install matplotlib
-            (mac)       $ sudo port install py27-matplotlib
-            (anything)  $ sudo pip install matplotlib
-            (anything)  $ conda install -c conda-forge matplotlib=2.0.0
-            or if you are working on a cluster, try loading a different python module...""")
-            sys.exit(1)
+            print("Failed to load matplotlib 2.*, trying 3.*")
+            try:
+                import matplotlib
```
Comment (acmnpv) [inline qscripts-cli/q_plot.py]: how does this code detect the other version of matplotlib? the code itself is identical?

File: qscripts-cli/q_plot.py [hunk 2]
```diff
@@ -90,8 +90,12 @@ def __init__(self, parent, plotdata, plotdata_files):
         self.canvas = FigureCanvasTkAgg(self.figure, master=self.parent)
         self.canvas.get_tk_widget().pack()
         self.canvas._tkcanvas.pack(fill=Tk.BOTH, expand=1)
-
-        self.toolbar = NavigationToolbar2TkAgg( self.canvas, self.parent )
+        
+        
+        if matplotlib.__version__[0] == "3":
```
Comment (acmnpv) [inline qscripts-cli/q_plot.py]: can this be instead > 2

File: qscripts-cli/q_plot.py (hunk 1 above)
Comment (FlorianvdEnt) [reply qscripts-cli/q_plot.py]: It doesn't it just tries to load it trought the old using the old way (NavigationToolbar2TkAgg) and if it doesn't work it loads it the 3.0.0+ way NavigationToolbar2Tk

File: qscripts-cli/q_plot.py [hunk 3]
```diff
@@ -394,21 +398,24 @@ def on_resize(self,event):
         try:
             import matplotlib
             matplotlib.use('TkAgg')
-            from matplotlib.backends.backend_tkagg import FigureCanvasTkAgg, NavigationToolbar2TkAgg
+            if int(matplotlib.__version__[0]) > 2:
```
Comment (acmnpv) [inline qscripts-cli/q_plot.py:401]: now blind people like myself can see that too, thanks!

**Pull request merged**